OsmAnd-java/src/main/java/net/osmand/router/ExitInfo.java

```diff
@@ -15,11 +16,19 @@ public void setRef(String ref) {
         this.ref = ref;
     }
 
-    public String getExitStreetName() {
-        return exitStreetName;
+    public String getDestinationRef() {
+        return destinationRef;
     }
 
-    public void setExitStreetName(String exitStreetName) {
```

Comment:
no need to delete exit name might be useful in future

Reply:
Ok